content/en/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes.md

```diff
@@ -0,0 +1,160 @@
+---
+title: Adding Windows worker nodes
+content_type: task
+weight: 50
+---
+
+<!-- overview -->
+
+This page explains how to add Windows worker nodes to a kubeadm cluster.
+
+## {{% heading "prerequisites" %}}
+
+* A running [Windows Server 2019](https://www.microsoft.com/en-us/cloud-platform/windows-server-pricing)
+(or higher) instance with administrative access.
+* If you are using VXLAN/Overlay networking you must have also have
+[KB4489899](https://support.microsoft.com/help/4489899) installed.
+* A running kubeadm cluster created by `kubeadm init` and following the steps
+in the document [Creating a cluster with kubeadm](/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/).
+* `curl.exe` is installed and present in the `PATH` environment variable.
+
+<!-- steps -->
+
+## Adding Windows worker nodes
+
+{{< note >}}
+To facilitate the addition of Windows worker nodes to a cluster, PowerShell scripts from the repository
+https://sigs.k8s.io/sig-windows-tools are used.
+{{< /note >}}
+
+Do the following for each machine:
+
+1. Connect to the machine by using SSH or another method.
+1. Make sure you are Administrator or a privileged user.
+1. Open a PowerShell session.
+
+Then proceed with the steps outlined below.
+
+### Install containerd
+
+Run the following commands to install containerd:
+
+  ```PowerShell
+  curl.exe -LO https://raw.githubusercontent.com/kubernetes-sigs/sig-windows-tools/master/hostprocess/Install-Containerd.ps1
+  .\Install-Containerd.ps1 -ContainerDVersion 1.7.1
```

Comment:
we should probably not recommend a specific version - i've seen this guidance in other areas of the K8s docs

Reply:
added 1.7.22 since IIUC there has to be a version here or it will use an older version as pinned inside the script. in this case 1.7.1, but we should probably decide what to do with this thread https://github.com/kubernetes/website/pull/47888#discussion_r1755223274

Reply:
It's OK to recommend a version provided we have the capacity to update this every time the other project does a release (we probably don't).

Reply:
reworked the steps to tell the user to pick a release version from the containerd repo.